omod/src/main/java/org/openmrs/module/pihcore/page/controller/account/ChangePasswordPageController.java

```diff
@@ -54,7 +54,7 @@ private String changePasswords(ChangePassword changePassword, UserService userSe
             request.getSession().setAttribute(EmrConstants.SESSION_ATTRIBUTE_INFO_MESSAGE,
                     messageSourceService.getMessage("emr.account.changePassword.success", null, Context.getLocale()));
             request.getSession().setAttribute(EmrConstants.SESSION_ATTRIBUTE_TOAST_MESSAGE, "true");
-        } catch (DAOException e) {
+        } catch (Exception e) {
             request.getSession().setAttribute(EmrConstants.SESSION_ATTRIBUTE_ERROR_MESSAGE,
```

Comment:
Catches more exceptions and displays as errors on the form, rather than barfing with a stack trace.